content/frontend-web/css-sass/webpack-configuration-for-sass-only.md

```diff
@@ -135,10 +143,9 @@ sass-webpack-project/
 │       └── style.css
 └── yarn.lock
 ```
-
-For the example code of this articles, you can checkout from [source code via Github](https://github.com/mildronize/web-starter-collection/tree/master/webpack5-sass-only-starter)
+For the example code of this article, you can checkout from [source code via Github](https://github.com/mildronize/web-starter-collection/tree/master/webpack5-sass-only-starter).
```

Comment:
Could you the send the PR  to this repo?, or you can ask me to do.

Reply:
I will make a PR later, for sure so you don't need to do it.

Reply:
@aaronamm Oh, thank, we can move this repo to **dotnetthailand** repo later